ui/index.html

```diff
@@ -1,12 +1,28 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <title>Hello World </title>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>

```

Comment:
Can you download all the .js and .css imports and put them in corresponding `js/` and `css/` folders? Loading them from the web makes it difficult to work offline.